src/exxhash/LICENSE.exxhash

```diff
@@ -0,0 +1,203 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
What's this license from/for?

Reply:
Good point. It's for the NIF itself vs (xxHash header). But we don't include apps separately any longer so the top level LICENSE would do there. I'll remove it.